src/resources/assets/js/components/settingsTabPlugin.vue

```diff
@@ -0,0 +1,32 @@
+<script>
+import {Events} from '../import';
+/**
+ * The plugin component to edit the Magic-SAM throttle interval.
+ *
+ * @type {Object}
+ */
+export default {
+    props: {
+        settings: {
+            type: Object,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            'SAMthrottleInterval': 1000,
+        };
+    },
+    watch: {
+        SAMthrottleInterval(interval){
+            Events.$emit('settings.SAMthrottleInterval', interval);
+            this.settings.set('SAMthrottleInterval', interval);
+        },
+    },
+    created() { 
+        if(this.settings.has('SAMthrottleInterval')){
```

Comment:
* Probably add another 0.2s option (medium-fast), which should work on all more or less recent hardware. 
* I also thought about "SAM speed", but it might be misleading as the speed of SAM is decreasing with the fast option as your CPU might build up load. The regular user will just turn it to fast as fast is always better. However, if you have a very old pc it might result in a slower experience for the user than on medium. SAM response time might be a more accurate term but nobody will get what that means I guess. So speed is probably still the best.
* What about making 0,5s the default as this should still work for everybody and people will feel the difference and probably will look for the settings. Otherwise most people will probably never find the option to make it faster.

Reply:
Then let's do "faster" (0.1 s), "fast" (0.2), "medium" (0.5), "slow" (1) and "slower" (2). Medium will be the new default.

What about "Magic SAM refresh rate" as label?